taskcluster/ci/ruff/kind.yml

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+kind-dependencies:
+    - docker-image
+
+# Copy of tox job, but doesn't use caching
+
+transforms:
+    - scriptworker_taskgraph.transforms.python_version:transforms
```

Comment:
Maybe we don't need these transforms? Would we ever want to run this on more than one Python version?